### ISSUE TITLE
[Feature Request] Clear queue when adding new album/playlist.

### ISSUE THREAD

**Issue by grgcnnr** [jaedb/Iris#134]
This is a common use case for us. When playing a new album we want to wipe the now playing queue and start fresh. 

Interested to know if anyone else finds this to be an issue or if I'm missing something.

Cheers.

**Comment by Informatic** [comment]
It may be slightly related... right now it seems that, when there are any tracks present in "Now playing", pressing "Play" on some Spotify playlist adds first song to the beginning of the queue, starts playing that, and then rest of songs in the end...

**Comment by grgcnnr** [comment]
yep @Informatic I'm seeing that too.
I reckon that's a legit bug and should probably have its own issue.
![untitled](https://user-images.githubusercontent.com/2694014/27411934-33ec105e-5745-11e7-99bf-425c3e40a73b.png)

**Comment by jaedb** [comment]
@grgcnnr and @Informatic correct, this is a separate issue to the initial Feature Request. Please open a new ticket for this problem.

**Comment by fekete-robert** [comment]
@grgcnnr: I think you are right in your original post, I'd also expect that starting a new playlist or album cleans the queue. So +1

**Comment by jaedb** [comment]
Currently when you play an album or playlist (ignoring the separate issue highlighted, now #136) all the tracks will play immediately. Once all tracks have played, then playback resumes with whatever was currently in your tracklist. Is this such a bad thing?

To me it's a nice way to retain a constant music experience.

**Comment by grgcnnr** [comment]
Definitely not a bad thing, a different preference at worst 😃.
What tends to happen in my house and office is this: you'll play so e nice music while your eating dinner, the album you put on will finish, then you'll get the second half of some gnarly prog rock you were listening to earlier and just paused at some stage.
If you tend to be the only person queuing songs or you run Iris 24/7 this might not be an issue.

Your call at the end of the day,  keen to see how others feel

**Comment by fekete-robert** [comment]
I agree with George :) It's nothing serious, just different from the
frontend we used earlier.
I mainly use mopidy in the office, and we daily start playing albums at
random, abandoning them if we don't like one. So after a while when one we
liked ends, another one we abandoned starts playing :)
Also, it is convenient to know that the end of the day (when the
album/playlist ends), music will stop playing, and not stay on for the
night.

(Also, many thanks for developing Iris, it's great to see a nice frontend
that's actively developed!)
